src/smart-components/role/add-role-new/add-permissions-template.js

```diff
@@ -30,9 +33,32 @@ const AddPermissionTemplate = ({ formFields }) => {
         Add permissions
       </Title>
       <TextContent>
-        <Text>Select permissions to add to your role</Text>
+        <Text>
+          Select permissions to add to your role
+          {unresolvedSplats.length !== 0 && (
+            <Popover
+              headerContent="Custom roles only support granular permissions"
+              bodyContent="Wildcard permissions (for example, approval:*:*) aren’t included in this table and can’t be added to your custom role."
```

Comment:
We should use an app that doesn't have granular permissions here, e.g. `patch:*:*`